cinn/optim/transform_gpu_forloop.cc

```diff
@@ -465,6 +466,114 @@ void OptimizeExprGPU(Expr *expr) {
       return x->As<ir::Store>();
     });
   }
+
+  // Add loops not bound by gpu axis in for_loop_info, used for recalculating the size of cache buffer after ComputeAt.
+  auto find_no_bind_forloops = ir::CollectIRNodesWithoutTensor(
+      *expr, [&](const Expr *x) { return x->As<ir::For>() && !x->As<ir::For>()->bind_info().valid(); });
+  for (auto &for_loop : find_no_bind_forloops) {
+    VLOG(3) << "format no bind loop:\n" << for_loop;
+  }
+
+  auto find_schedule_block_realize =
+      ir::CollectIRNodesWithoutTensor(*expr, [&](const Expr *x) { return x->As<ir::ScheduleBlockRealize>(); });
+  for (auto &schedule_block_expr : find_schedule_block_realize) {
+    auto schedule_block_realize            = schedule_block_expr.As<ir::ScheduleBlockRealize>();
+    auto schedule_block                    = schedule_block_realize->schedule_block.As<ir::ScheduleBlock>();
+    auto iter_vars                         = schedule_block->iter_vars;
+    auto iter_values                       = schedule_block_realize->iter_values;
+    auto compute_at_extra_var_iter         = schedule_block->attrs.find("compute_at_extra_var");
+    auto reverse_compute_at_extra_var_iter = schedule_block->attrs.find("reverse_compute_at_extra_var");
```

Comment:
```suggestion
    auto compute_at_extra_var_iter         = schedule_block->attrs.find(compute_at_extra_var);
```
用字面常量替换`constexpr const char* compute_at_extra_var = "compute_at_extra_var"`

Reply:
好的，将在下一个pr中修改